environment_raven.yml

```diff
@@ -0,0 +1,29 @@
+name: synth_seg_py39
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9.7
```

Comment:
Is there a reason for not using Python 3.10? It has `match` which is finally something similar to a switch statement :)

Reply:
3.9.7 is the most up-to-date python environment we have readily installed on our systems, so I would stick to it. But I will ask around if there are plans in the near future to also provide a python 3.10 env.